lib/airbrake.js

```diff
@@ -10,6 +10,7 @@ var stackTrace = require('stack-trace');
 var hashish = require('hashish');
 var querystring = require('querystring');
 var stringify = require('json-stringify-safe');
+var execSync = require('sync-exec');
```

Comment:
@shime Maybe there needs to be a support section to the README so people know what versions `node-airbrake` supports.